### DISCUSSION TITLE
Add RATE financial function

### DISCUSSION THREAD

**Discussion by snhv** [Qalculate/libqalculate#391] [Ideas]
Hi,
We already have a bunch of financial functions such as PV, FV, EFFECTIVE, NOMINAL, but not RATE, another essential function in the pack.
![image](https://user-images.githubusercontent.com/22385284/152551188-763668b5-5422-460e-83cb-bb4b3e0a4a51.png)
Could we have RATE please?
Cheers.

**Comment by hanna-kn** [comment]
Done (solves pmt(x, ...)=pmt using newtonsolve, with the function expression `newtonsolve(pmt("x",\x,\z,\A{0},\B{0})=\y,\C{0.1},"x")`).

**Comment by snhv** [comment]
Thank you so much!
